### DISCUSSION TITLE
Improve more Sanny Builder syntax.

### DISCUSSION THREAD

**Discussion by XMDS** [sannybuilder/dev#235] [General]
cleo redux uses JavaScript to write Cleo scripts, which is very advanced. I have noticed that many people are already using it, but we should still improve and optimize more Sanny syntax, which is important. In cross-platform GTA games, such as PS2, PSP, Android, iOS, etc., their Cleo libraries still use .cs scripts, and many people still use Sanny Builder tool's opcode form to write Cleo scripts. Therefore, we should optimize this Sanny language even more.

**Comment by XMDS** [comment]
Feel free to discuss and suggest more ways to write commands (opcodes), such as abbreviations, keywords, more syntax symbols, etc.

**Comment by x87** [comment]
I have a specific filter to display all proposals
https://github.com/sannybuilder/dev/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Language+Specification%22

**Comment by x87** [comment]
moving to Discussions
